src/transaction.js

```diff
@@ -177,6 +177,8 @@ Transaction.prototype.clone = function () {
   return newTx
 }
 
+var one = new Buffer('0000000000000000000000000000000000000000000000000000000000000001', 'hex')
```

Comment:
Is the current so bad?  Its not like this hex is parsing at run time?
On 4 May 2015 2:14 am, "Wei Lu" <notifications@github.com> wrote:

> In src/transaction.js
> <https://github.com/bitcoinjs/bitcoinjs-lib/pull/400#discussion_r29557399>
> :
>
> > @@ -177,6 +177,8 @@ Transaction.prototype.clone = function () {
> >    return newTx
> >  }
> >
> > +var one = new Buffer('0000000000000000000000000000000000000000000000000000000000000001', 'hex')
>
> Really? How about
>
> var ONE = new Buffer(64)
> ONE.fill(0)
> ONE[63] = 1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoinjs/bitcoinjs-lib/pull/400/files#r29557399>.
>